r/tests/testthat/test-dplyr-mutate.R

```diff
@@ -415,3 +415,160 @@ test_that("mutate and write_dataset", {
       summarize(mean = mean(integer))
   )
 })
+
+# PACHA ADDITIONS ----
+# READ THIS CAREFULLY PLEASE, IT'S MY 1ST DAY WRITING THIS KIND OF SENSITIVE TESTS
+
+# similar to https://github.com/tidyverse/dplyr/blob/master/tests/testthat/test-mutate.r#L1-L10
+# the rest of that test belongs in L55-62 here
```

Comment:
Please delete the existing test at L55-62 and another line in this test to check empty `%>% mutate()` on `tbl`